Invoke-SqlCmd2/Public/Invoke-SqlCmd2.ps1

```diff
@@ -60,6 +60,10 @@ function Invoke-Sqlcmd2 {
         .PARAMETER AppendServerInstance
             If this switch is enabled, the SQL Server instance will be appended to PSObject and DataRow output.
 
+        .PARAMETER ParseGo
```

Comment:
Hiyo!  One more question : )  Would it be worth mentioning that this splits a query into multiple queries if it sees GO?  Folks might not take that away if the phrasing is `stripped away`?

Cheers!